spec/models/pglogical_subscription_spec.rb

```diff
@@ -377,9 +377,7 @@ def with_an_invalid_remote_schema
       expect(remote_region.auth_key_configured?).to be true
 
       sub.delete
-      remote_region.reload
-
-      expect(remote_region.auth_key_configured?).to be false
+      expect(AuthToken.find_by_id(auth.id)).to be_nil
```

Comment:
It actually does a few things.

It removes all that region's data from the database, removes the authentication record for that region, and also removes the pglogical configuration if that was the last subscription.

I can change these tests up a bit to make that more clear.